CIScripts/Test/TestRunner.ps1

```diff
@@ -43,10 +43,7 @@ function Invoke-TestScenarios {
 
     $TestsBlacklist = @(
         # Requires Docker Driver
-        "vRouterAgentService.Tests.ps1",
-
-        # Requires Agent
-        "vRouterAgentMSIInstaller.Tests.ps1"
+        "vRouterAgentService.Tests.ps1"
```

Comment:
It could be removed from blacklist, as you've changed it not to require docker driver.